packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.Application.Example.tsx

```diff
@@ -45,6 +46,12 @@ export class ShimmerApplicationExample extends BaseComponent<{}, IShimmerApplica
       columns: _buildColumns(),
       isDataLoaded: false
     };
+
+    this._async = new Async(this);
+  }
+
+  public componentWillMount(): void {
```

Comment:
@Vitalius1 you'll want to dispose in `componentWillUnmount` not `componentWillMount` as only the former occurs when removing the element from the DOM.

```suggestion
  public componentWillUnmount(): void {
```

Reply:
😄 good catch i didn’t notice it how auto completed the method name.